ci/setupEnv.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Is this one used?
For now, we can use test certificates
Maybe we can extend docker compose with omejdn and use test certs for that one and not Fraunhofer ones.

Reply:
God that you noticed, I forgot to remove it, since we are now using actions instead of doing it manually like this. As for omejdn, it's a good idea and I agree. We can add it as a separate task.

Reply:
Please create ticket in backlog so we do not forget it

Reply:
Done